src/core/src/package_managers/YumPackageManager.py

```diff
@@ -72,6 +79,14 @@ def invoke_package_manager(self, command):
         """Get missing updates using the command input"""
```

Comment:
just for my knowledge, is this comment or logging?

Reply:
comment